apps/mmclient/mattermost_client_pp.go

```diff
@@ -52,11 +58,19 @@ type ClientPP struct {
 
 	// FalseString is the string value sent to the server for false boolean query parameters.
 	falseString string
+
+	fromPlugin bool
```

Comment:
I don't see `fromPlugin` being set anywhere - is it needed?

Reply:
It's used in https://github.com/mattermost/mattermost-plugin-apps/blob/722b4066c8b9b27d1fab06239eca4475b8ca495b/apps/mmclient/mattermost_client_pp.go#L239-L241

to build the base URL.

Reply:
It's used in https://github.com/mattermost/mattermost-plugin-apps/blob/722b4066c8b9b27d1fab06239eca4475b8ca495b/apps/mmclient/mattermost_client_pp.go#L237-L244
to build the base URL.